pkg/skaffold/trigger/triggers.go

```diff
@@ -220,3 +220,21 @@ func (t *apiTrigger) Debounce() bool {
 func (t *apiTrigger) WatchForChanges(out io.Writer) {
 	color.Yellow.Fprintln(out, "Watching on designated port for build requests...")
 }
+
+// StartTrigger attempts to start a trigger.
+// It will attempt to start as a polling trigger if it tried unsuccessfully to start a notify trigger.
+func StartTrigger(ctx context.Context, t Trigger) (<-chan bool, error) {
+	ret, err := t.Start(ctx)
+	if err != nil {
```

Comment:
Can you invert the `if` to first handle the error case?

Reply:
sure